### PR TITLE
Add "Try Console" selector to the Console Widget

### PR DESCRIPTION
**Pull request by scottybollinger**
Updates the "View Console" link to say "Try Console" and adds a selector to the Console Widget, when the server cannot be reached.

image

[Example](https://docs_bk_2974.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/current/getting-started.html#add-single-document)

Demo:

https://github.com/elastic/docs/assets/1869731/ac627840-9653-4683-8e33-e23ce8c2a226

### CODE REVIEW

File: resources/web/docs_js/components/console_widget.jsx [hunk 1]
```diff
@@ -62,6 +62,56 @@ export class _ConsoleForm extends Component {
...
+
+    return (
+      <div className="try_console_selector">
+        <h4>Try in Console</h4>
+        <p>We were unable to detect a running console server.</p>
+        <p>
+          <a
+            id="try_console_selector_try_cloud_button"
+            className="button btn-primary btn-small"
+            href="https://cloud.elastic.co/registration"
```
Comment (bmorelli25) [inline resources/web/docs_js/components/console_widget.jsx:86]:
```suggestion
            href="https://www.elastic.co/cloud/elasticsearch-service/signup?page=docs&placement=console-signup"
```

@KOTungseth do we still track signups with the `page=docs` query param? ([source](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L188))

File: resources/web/docs_js/components/console_widget.jsx [hunk 2]
```diff
@@ -62,6 +62,56 @@ export class _ConsoleForm extends Component {
...
+            target="_blank"
+          >
+            Sign up for Elastic Cloud trial
```
Comment (bmorelli25) [inline resources/web/docs_js/components/console_widget.jsx:89]:
```suggestion
            Start a free trial of the Elasticsearch Service
```
Is this too long? I'm looking at our [terminology guide](https://docs.google.com/document/d/1GD50O8BgOAAp9OhJTdouISEfrpu5VRww4lvX4pom9QY/edit).

File: resources/web/docs_js/components/console_widget.jsx [hunk 3]
```diff
@@ -62,6 +62,56 @@ export class _ConsoleForm extends Component {
...
+          </a>
+        </p>
+        <p>
+          <a
+            id="try_console_selector_install_elasticsearch_button"
+            href="https://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html"
```
Comment (bmorelli25) [inline resources/web/docs_js/components/console_widget.jsx:95]:
```suggestion
            href="https://www.elastic.co/guide/en/elastic-stack/current/installing-elastic-stack.html"
```

I think this is a better link

File: resources/web/docs_js/components/console_widget.jsx (hunk 1 above)
Comment (KOTungseth) [reply resources/web/docs_js/components/console_widget.jsx:86]: Nope. I don't the mechanism that tracks this data.

File: resources/web/docs_js/components/console_widget.jsx (hunk 2 above)
Comment (KOTungseth) [reply resources/web/docs_js/components/console_widget.jsx:89]: We are all over the board here. We never say `Elasticsearch Service`, but instead use `Cloud`. 

Let's use:

`Start a free Cloud trial`

File: resources/web/docs_js/components/console_widget.jsx [hunk 4]
```diff
@@ -62,6 +62,56 @@ export class _ConsoleForm extends Component {
...
+            Install Elasticsearch and Kibana locally
+          </a>
+        </p>
+        <p>
+          <a
+            id="try_console_selector_configure_example_widget_button"
+            href="#"
+            onClick={handleConfigureClick}
+          >
+            Configure the example widget
```
Comment (KOTungseth) [inline resources/web/docs_js/components/console_widget.jsx:107]:
```suggestion
            Configure Console settings
```

**Review by KOTungseth — DISMISSED** [review]
Can we change this window?

Screenshot 2024-04-01 at 3 45 21 PM

File: resources/web/docs_js/components/console_widget.jsx (hunk 1 above)
Comment (bmorelli25) [reply resources/web/docs_js/components/console_widget.jsx:86]: This is (was?) something finance or marketing used. I can't remember.

File: resources/web/docs_js/components/console_widget.jsx (hunk 2 above)
Comment (bmorelli25) [reply resources/web/docs_js/components/console_widget.jsx:89]: From [the terminology guide](https://docs.google.com/document/d/1GD50O8BgOAAp9OhJTdouISEfrpu5VRww4lvX4pom9QY/edit):

> Preferred terms are “on Elasticsearch Service” or “on Elastic Cloud” if there is space. If not “on cloud” is OK. 

Should we use Elastic Cloud instead?

Comment (bmorelli25) [reply resources/web/docs_js/components/console_widget.jsx:89]: > We never say Elasticsearch Service, but instead use Cloud.

Where is this guidance from btw? This goes against the terminology recommendations that I know.

**Comment by scottybollinger** [comment]
@KOTungseth it would not let me reply to this comment. Is [this](https://cloud.elastic.co/registration) the link you want to use here?

image

**Comment by KOTungseth** [comment]
> @KOTungseth it would not let me reply to this comment. Is [this](https://cloud.elastic.co/registration) the link you want to use here?
> 
> image

Yes please

**Comment by KOTungseth** [comment]
> > Can we change this window?
> > Screenshot 2024-04-01 at 3 45 21 PM
> 
> What changes do you want, exactly?

`Enter the URL of the Console editor` -> `Console URL`

`Or install [Kibana](https://www.elastic.co/guide/en/kibana/master/setup.html).` -> `For information on how to set up and run Kibana, check [Set up](https://www.elastic.co/guide/en/kibana/current/setup.html).

**Comment by scottybollinger** [comment]
[Updated](https://github.com/elastic/docs/pull/2974/commits/666094ec46adf824ff31489b2d6f98f07d9fa076)

image

**Review by bmorelli25 — APPROVED** [review]

File: resources/web/docs_js/components/console_widget.jsx (hunk 1 above)
Comment (KOTungseth) [reply resources/web/docs_js/components/console_widget.jsx:131]: This was owned by CSG, but the specific people who tracked this are gone and did not pass off the information. I've asked others, but they are also unable to find the information.

File: resources/web/docs_js/components/console_widget.jsx (hunk 2 above)
Comment (KOTungseth) [reply resources/web/docs_js/components/console_widget.jsx]: `Start a free Elastic Cloud trial` works for me. 

I'm working from the messaging on https://www.elastic.co/cloud.

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**